plugin-security.policy

```diff
@@ -37,7 +37,6 @@ grant {
   permission java.util.PropertyPermission "*","read,write";
```

Comment:
This permissions covers read and write for all properties so anything else is redundant

Reply:
Can we investigate these other policy lines separately to minimize the surface area of this change?

Reply:
The only PropertyPermission this code change removes is `permission java.util.PropertyPermission "org.apache.xml.security.ignoreLineBreaks", "write";`, but this explicit permission isn't required since there's already an item for `permission java.util.PropertyPermission "*","read,write";` near the top of this file. I removed another permission that was commented out as well.